README.md

```diff
@@ -65,6 +65,8 @@ To build,
 4. Install javascript dependencies: `npm install`.
 5. Build the client: `npm run build`.
 
+The client can also be built with `yarn`. From `cd octree_web_viewer/client` use the commands `yarn install` and `yarn build`.
```

Comment:
Rather than adding it here I would replace it in `4.`/`5.` above:
```
4. Install javascript dependencies: `yarn install`.
5. Build the client: `yarn build`.
```

Reply:
done!